perfkitbenchmarker/linux_benchmarks/spark_benchmark.py

```diff
@@ -67,6 +74,9 @@
                   'output of the job')
 flags.DEFINE_list('spark_job_arguments', [], 'Arguments to be passed '
                   'to the class given by spark_classname')
+flags.DEFINE_enum('spark_job_type', spark_service.SPARK_JOB_TYPE,
```

Comment:
The prefix of the name is the name of the benchmark,  so that's why `spark_job_type`.  This is unfortunately named, I agree; I wouldn't mind massive renaming if I could figure out better names.